.github/workflows/pull_request.yml

```
@@ -0,0 +1,36 @@
+  pull_request:
+    branches:
+      - development
+
+jobs:
+  prettier-lint:
+    name: Prettier and Lint
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
```

actionlint (1.7.3)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)